runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -408,43 +397,27 @@ MM_StandardAccessBarrier::jniReleasePrimitiveArrayCritical(J9VMThread* vmThread,
 {
 	J9JavaVM *javaVM = vmThread->javaVM;
 	J9InternalVMFunctions *functions = javaVM->internalVMFunctions;
```

Comment:
I feel like this is not used any more here (but only in new helper methods) or any other get/release method. If so, we should not have it here and then not to pass it, but let the helper fetch it.